src/Amazon.SecretsManager.Extensions.Caching/Amazon.SecretsManager.Extensions.Caching.csproj

```diff
@@ -7,7 +7,7 @@
         <OutputType>Library</OutputType>
         <DisableImplicitNamespaceImports>true</DisableImplicitNamespaceImports>
         <PackageId>AWSSDK.SecretsManager.Caching</PackageId>
-        <PackageVersion>1.0.6</PackageVersion>
+        <PackageVersion>1.0.7</PackageVersion>
```

Comment:
Can we start doing proper semver? Yeah we are still `netstandard2.0`, but we do upgrade the `Microsoft.Extensions.Caching.Memory` from `7` to `8`. This is not a seamless patch for our consumers. Suddenly they'll might need to use the binding redirect.

At minimum we should do the `2.0.0` so people who accept new features and rely on API stability (e.g. `^1`) wont get affected. 

Since `Microsoft.Extensions.Caching.Memory` is our only "real" dependency. What do you think about following the AspNet schema with their versioning? E.g. now it would be 8, and we would need to relax the dependency to memory on something like `^8.0`

Also our doc is going to get out of sync ([README.md](https://github.com/aws/aws-secretsmanager-caching-net/blob/master/README.md?plain=1#L31)). Do we want to use caret or star there as well?

Reply:
That's fair, we can move to `2.0.0`.

I'll update the README to pull `2.*`.

I can relax the `Microsoft.Extensions.Caching.Memory` to `8.x`. That comes with the risk that Microsoft might release a non-semver compliant version (ex. breaking change in minor/patch update) and break customers. The benefit is that customers get bug fixes that come with newer patch and minor versions.

It's worth mentioning that `8.0.0` is actually a version range `>= 8.0.0` with a preference for lowest version that meets the criteria.